### ISSUE TITLE
[GUI] Small values are rounded to zero when hovering a point in a graph

### ISSUE THREAD

**Issue by merkste** [prismmodelchecker/prism#207]
It appears that small values are always rounded to 3 decimal places. Maybe we should switch to scientific notation then?

**Comment by davexparker** [comment]
Yes, it could make sense to fix that, although I don't immediately see whether that is done in PRISM code or in JFreeChart code.
